src/pudl/metadata/resources/eia930.py

```diff
@@ -192,6 +192,31 @@
         "etl_group": "eia930",
         "create_database_schema": False,
     },
+    "out_eia930__hourly_demand": {
+        "description": (
+            """EXPERIMENTAL / WORK-IN-PROGRESS, 2025-04-04.
+
+This aggregates imputed demand from the ``out_eia930__hourly_operations`` table.
+It aggregates demand to the level of EIA regions, interconnects, and the contiguous United States."""
+        ),
+        "schema": {
+            "fields": [
+                "datetime_utc",
+                "code",
+                "aggregation_group",
+                "demand_agg_mwh",
+            ],
+            "primary_key": [
+                "datetime_utc",
+                "code",
```

Comment:
What does code mean here? That's awfully generic. Need to explain the table description, or use a more descriptive column label.